lib/longtail/stackoverflow/posts.pl

```diff
@@ -75,7 +75,8 @@
                 # 1 = question, 2 = answer, 3 might be external
                 my $link_type_id = $4;
                 #warn "$id -> $post_id -> $related_post_id -> $link_type_id\n";
-                push (@{$post_links{$post_id}} ,$related_post_id);
+                $post_links{$post_id}{$related_post_id} = 1;
+                $post_links{$related_post_id}{$post_id} = 1;
```

Comment:
Still need to change how `post_links` is added to `meta` below.  If there are links it's currently sent through as a hash:

> +<field name="meta">{"post_links":{"218005":1,"29665":1,"275036":1,"81866":1,"9766":1,"1737":1}

Either set to an empty hash if no links or do:
```Perl
post_links => [keys %{$post_links{$parent_id}}],
```
Unless you want each link count now the latter is probably better until we have a use for the counts.